Cargo.lock

```diff
@@ -1183,6 +1183,11 @@ dependencies = [
  "typenum 1.10.0 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "indexmap"
```

Comment:
Does this data structure have any "classic" name?
Search by "IndexMap" only leads to the Rust version.

Reply:
I don't really know (for the record, this crate used to be called `ordermap`). cc @bluss